src/config.rs

```diff
@@ -27,7 +27,7 @@ impl Config {
             .set_rest_api_endpoint("https://testnet.binance.vision")
             .set_ws_endpoint("wss://testnet.binance.vision")
             .set_futures_rest_api_endpoint("https://testnet.binancefuture.com")
-            .set_futures_ws_endpoint("wss://testnet.binancefuture.com")
+            .set_futures_ws_endpoint("wss://fstream.binancefuture.com")
```

Comment:
According to this post : https://dev.binance.vision/t/websocket-url-for-user-data-stream-on-testnet/13085
testnet for future streams is at stream.binancefuture.com and the fstream is for production. do you confirm ?

Reply:
If so, change it to stream.binancefuture.com for testnet and I'll merge

Reply:
2024-01-24's update of https://binance-docs.github.io/apidocs/futures/en/#change-log states wss://fstream.binancefuture.com is correct, now both are usable, but wss://stream.binancefuture.com may be deprecated in the future.
![Screenshot_2024-02-05_23-39-40](https://github.com/Igosuki/binance-rs-async/assets/6938607/e35ed93c-3930-43cd-b08b-b07c1083d4a7)

Reply:
Ok